python/tvm/relay/op/strategy/cuda.py

```diff
@@ -224,7 +231,7 @@ def conv2d_strategy_cuda(attrs, inputs, out_type, target):
             if use_auto_scheduler and judge_winograd_auto_scheduler:
                 strategy.add_implementation(
                     wrap_compute_conv2d(topi.nn.conv2d_winograd_nhwc),
-                    wrap_topi_schedule(tvm.te.create_schedule),
```

Comment:
`tvm.te.create_schedule` is not a valid topi schedule. Their signatures are different